src/index.html

```diff
@@ -2,13 +2,52 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name = "viewport"
+          content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    npmcrossorigin=" ">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#index.html" class="header__link">
+        <img src="images/logo.png"
+        class="logo" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#sumsung" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">SMARTPHONES</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops&computers" class="nav__link"
+                  data-qa="hover">LAPTOPS & COMPUTERS</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#sumsung" class="nav__link">
              SAMSUNG
              </a>
          </li>
          
          <li class="nav__item">
            <a href="#smartphones" class="nav__link">SMARTPHONES</a>
          </li>
          
          <li class="nav__item">
            <a 
              href="#laptops&computers" 
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
```
add empty line between siblings (but not between parent and child)
if tag has more than two attributes - each should be on new line.
uppercase better to make with styles